packages/components/src/date-time/types.ts

```diff
@@ -61,6 +61,11 @@ export type TimeInputProps = {
 	 */
 	minutesProps?: React.ComponentProps< typeof MinutesInput >;
 
+	/**
+	 * The label for the time input.
+	 */
+	label?: string;
```

Comment:
Should we consider making the `label` required by default, so that when the input is used standalone consumers are nudged towards labelling it? We could disable the rule when compositing is in `DateTimePicker`.

Reply:
I don't think it's a huge issue for this one specifically, because the hours and minutes inputs are both labeled.

In the general case though (re: #51740), I'm starting to think that an eslint rule for our repo might be a good way to nudge devs without compromising the "truthfulness" of our TS types.